server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/CompactableImpl.java

```diff
@@ -973,14 +973,15 @@ public ExternalCompactionJob reserveExternalCompaction(CompactionServiceId servi
       Map<String,String> overrides =
           CompactableUtils.getOverrides(job.getKind(), tablet, cInfo.localHelper, job.getFiles());
 
-      var newFile = tablet.getNextMapFilename(!cInfo.propagateDeletes ? "A" : "C");
-      var compactTmpName = new TabletFile(new Path(newFile.getMetaInsert() + "_tmp"));
+      String tmpFileName =
+          tablet.getNextMapFilename(!cInfo.propagateDeletes ? "A" : "C").getMetaInsert() + "_tmp";
+      TabletFile compactTmpName = new TabletFile(new Path(tmpFileName));
```

Comment:
Could have a little static function in CompactableUtils that does this, then call that function here and in CompactableUtils.  That makes it easier for anyone looking at the code to find the two places where this happens.

Reply:
Moved to a utility function in Tablet in 9d1373b